students/18th/team2/soojinlee/user/views.py

```diff
@@ -0,0 +1,91 @@
+import json
+import bcrypt
+import jwt
+
+
+from django.http  import JsonResponse, HttpResponse
+from django.views import View
+from django.db.models import Q
+
+from .models      import User
+from my_settings    import SECRET_KEY, ALGORITHM
```

Comment:
너무 깔끔한데 조금 만 더 깔끔하게 만들어보자면 아래처럼 변경해주실 수 있겠네요!
```suggestion
import json
import bcrypt
import jwt

from django.http      import JsonResponse, HttpResponse
from django.views     import View
from django.db.models import Q

from .models     import User
from my_settings import SECRET_KEY, ALGORITHM
```

Reply:
수정하겠습니다~!🔥